requirements.txt

```diff
@@ -1,9 +1,10 @@
-Django==1.8.2
-django_compressor
 django-appconf
 django-ses-backend
-psycopg2==2.6
+Django==1.8.2
+django_compressor
 djmail
+futures
```

Comment:
i alpha-sorted these, futures is the backport from Python3's concurrent package to Python2

also, you should consider bumping up the Django and psycopg2 versions